impress-org/givewp
# Deprecate give_currencies filter hook

## Description

Added `@deprecated` tag to filter hook and add guidelines to register a new currency. 

## Affects

It does not affect any functionality.

## Testing Instructions

This pull request adds comment, so no need to test ti, just verify comment content.

## Pre-review Checklist
